docs/en/UM_Authentication.md

```diff
@@ -9,30 +9,40 @@ layout: page
 
 To maintain the security of your applications and users, all requests to Adobe I/O APIs must be authenticated and authorized using standards such as OAuth and JSON Web Tokens (JWT). Before you can access the User Management API, you will need to obtain access credentials by creating a new Integration in the Adobe I/O Console or Admin Console.
 
-![Diagram of Authentication workflow](AuthWorkflow.png)
 
 ## One-time Setup
 
-To obtain the credentials you need to access the User Management service, create a **Service Account Integration** using the [Adobe I/O Console](https://console.adobe.io/).
+To obtain the credentials you need to access the User Management service, create a **Project** using the [Adobe Developer Console](https://developer.adobe.com/).
 
-* A member of the organization with admin rights can create the Integration. You can use the Admin Console to grant administrative privilege to users.
-* You will need to create or purchase a digital signing certification, and use it to sign a JSON Web Token (JWT), which you will use to authenticate your requests.
+* Only members of the organization with SYSTEM ADMIN role can create the Integration for UMAPI. You can use the Admin Console to grant administrative privilege to users.
```

Comment:
I thought you could create them with developer role?

Reply:
No, it's available only to System admins. Any other account would see it greyed out.